pynicotine/gtkgui/ui/dialogs/about.ui

```diff
@@ -209,25 +209,27 @@ Attributions
  - daelstorm
```

Comment:
Line 206 > English
 - mathiascode (2020`-2022`)

Reply:
Are you sure, they are missing both at the start and at the end. It looks nicer in the code, alright.

Reply:
The problem is because this section lacks a title such as "Nicotine+ Contributors"